src/ace/managers/viewport/simplebuffer.c

```diff
@@ -187,7 +187,7 @@ tSimpleBufferManager *simpleBufferCreate(
 	}
 
 	simpleBufferSetFront(pManager, pFront);
-	simpleBufferSetBack(pManager, pBack == 0 ? pFront : pBack);
+	simpleBufferSetBack(pManager, pBack ? pFront : pBack);
```

Comment:
Ommiting `== 0` actually flips the logic.

Reply:
fixed in 6301206